test/NosCore.Tests/MapperTests.cs

```diff
@@ -42,30 +47,30 @@ namespace NosCore.Tests
     [TestClass]
     public class MapperTests
     {
-        private static readonly ILogger _logger = Logger.GetLoggerConfiguration().CreateLogger();
-        private readonly IGenericDao<AccountDto> _accountDao = new GenericDao<Account, AccountDto, long>(_logger);
+        private static readonly ILogger Logger = Core.I18N.Logger.GetLoggerConfiguration().CreateLogger();
+        private readonly IGenericDao<AccountDto> _accountDao = new GenericDao<Account, AccountDto, long>(Logger);
         private readonly MapsterMapper.Mapper _mapper = new MapsterMapper.Mapper();
 
         private readonly IGenericDao<CharacterDto> _characterDao =
-            new GenericDao<Character, CharacterDto, long>(_logger);
+            new GenericDao<Character, CharacterDto, long>(Logger);
 
-        private readonly IGenericDao<IItemInstanceDto> _itemInstanceDao = new ItemInstanceDao(_logger);
+        private readonly IGenericDao<IItemInstanceDto> _itemInstanceDao = new ItemInstanceDao(Logger);
 
         [TestInitialize]
         public void Setup()
         {
             TypeAdapterConfig<CharacterDto, GameObject.Character>.NewConfig().ConstructUsing(src =>
-                new GameObject.Character(null, null, null, _characterDao, _itemInstanceDao, null, _accountDao, _logger,
-                    null, null, null, null, null));
+                new GameObject.Character(new Mock<IInventoryService>().Object, new Mock<IExchangeProvider>().Object, new Mock<IItemProvider>().Object, _characterDao, _itemInstanceDao, new Mock<IGenericDao<InventoryItemInstanceDto>>().Object, _accountDao, Logger,
+                    new Mock<IGenericDao<StaticBonusDto>>().Object, new Mock<IGenericDao<QuicklistEntryDto>>().Object, new Mock<IGenericDao<MinilandDto>>().Object, new Mock<IMinilandProvider>().Object, new Mock<IGenericDao<TitleDto>>().Object));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 245 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)